### DISCUSSION TITLE
ONS configuration error on oracle connections

### DISCUSSION THREAD

**Discussion by JordiMontane** [dbeaver/dbeaver#8926] [Q&A] [answered]
#### System information: 
- Operating system. MacOS 10.15.4
- DBeaver version. 7.1 CE
- Java version 8

#### Connection specification:
- Database name and version. Oracle database
- Driver name. oracle.jdbc.OracleDriver [12.2.0.1], ojdbc8.jar
- Do you use tunnels or proxies (SSH, SOCKS, etc)? NO

#### Describe the problem you're observing:
since upgrading to beaver 7.1 my existing Orcale connections throw an error when starting the connection:
Unexpected driver connection error
ONS configuration
ONS configuration
timeout

the existing  prostgres connections work ok as with previous version (7.0.5).
#### Steps to reproduce, if exist:
double click on connection or highlight the connection and click the 'connect' icon

The error log does not show any entry.

**Comment by JordiMontane** [comment]
from what I see it is more related to the ojdbc  drivers that the dbeaver upgrade demanded than to  the dbeaver itself.
I'll downgrade to ojdbc7.jar to check

**Comment by JordiMontane** [comment, marked as answer]
OK. downgrading to ojdbc7.jar the connections work ok.

**Comment by TuSKan** [comment]
Same error

**Comment by p91paul** [comment]
see https://github.com/dbeaver/dbeaver/issues/9002 for a workaround
